config/config.go

```diff
@@ -78,28 +77,24 @@ type Benchmarks struct {
 	CisEks []string `config:"cis_eks,omitempty" yaml:"cis_eks,omitempty" json:"cis_eks,omitempty"`
 }
 
-var DefaultConfig = Config{
-	Period: 4 * time.Hour,
+var DefaultConfig = AgentInput{
+	Type: InputTypeEks,
+	Streams: []Stream{{
+		Period: 4 * time.Hour,
+	}},
 }
 
 func New(cfg *config.C) (Config, error) {
 	c := DefaultConfig
 
 	if err := cfg.Unpack(&c); err != nil {
-		return c, err
-	}
-
-	return c, nil
-}
-
-// GetActivatedRules returns the activated rules from the config.
-func (c *Config) GetActivatedRules() (*Benchmarks, error) {
-	cfgStream := c.Streams
-	if len(cfgStream) == 0 || cfgStream[0].RuntimeCfg == nil {
-		return nil, fmt.Errorf("could not find runtime cfg")
+		return Config{}, err
 	}
 
-	return cfgStream[0].RuntimeCfg.ActivatedRules, nil
+	return Config{
+		Stream: c.Streams[0],
```

Comment:
Thanks @ofiriro3, Such a case couldn't happen because we use `DefaultConfig` to construct the new configuration.

```
var DefaultConfig = AgentInput{
	Type: InputTypeVanillaK8s,
	Streams: []Stream{{
		Period: 4 * time.Hour,
	}},
}
```

If the agent config will not have a `streams` attribute, the new config will hold the default one.

Regarding `cfg.runtime`, no runtime config will result in running all rules and it's being validated in `opa.go`.